Scripts/script-ReadFile.yml

```diff
@@ -0,0 +1,33 @@
+commonfields:
+  id: eccea40d-921c-4a52-875a-738855531f58
+  version: -1
+name: ReadFile
+script: |-
+  res = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryID']})
+  with open(res[0]['Contents']['path'], 'r') as f:
+      data = f.read()
+  if data:
+      message = 'Read %d bytes from file.' % (len(data))
+      demisto.results({"Type" : entryTypes["note"],
+                       "ContentsFormat" : formats["text"],
+                       "Contents" : message,
```

Comment:
Put the data on the Contents, so one can extend the context with value fetched from file to other places